### ISSUE TITLE
weekly frequency recurrences under FireFox

### ISSUE THREAD

**Issue by gso** [jakubroztocil/rrule#87]
Weekly frequency rules are returning recurrences 1 day ahead of the date they should recur on when running under OPenBSD FireFox ESR 31.5.0, running the same rule under Chrome and the correct result is given https://stackoverflow.com/questions/29150105/odd-rrule-result

**Comment by jakubroztocil** [comment]
I'm getting this result in Firefox:
```
1	Mon	Mar	23	2015	00:03:54	GMT+0100	(CET)
2	Mon	Mar	30	2015	00:03:54	GMT+0200	(CEST)
```

http://jakubroztocil.github.io/rrule/#/rfc/FREQ=WEEKLY;COUNT=2;WKST=MO;BYDAY=MO

**Comment by gso** [comment]
Same result:
    1	Sun 	Mar 	22 	2015 	03:43:17 	GMT+0000 	(GMT)
    2	Sun 	Mar 	29 	2015 	03:43:17 	GMT+0100 	(BST)
How about FireFox  ESR https://www.mozilla.org/en-US/firefox/organizations/all/, I am also running an OpenBSD stable install, SSH socks 5h, the usual noscript etc. addons (github whitelisted), however the result does not change in safe mode with addons disabled.  As reported by the test page: Mozilla/5.0 (X11; OpenBSD i386; rv:31.0) Gecko/20100101 Firefox/31.0  I will check if someone else can duplicate this on OpenBSD.

**Comment by gso** [comment]
A reinstall fixed this, apols for the false alarm.  Usually if there is a problem with a hacker the machine is trashed more or less immediately -- we've had quite a few security fixes recently, my guess is an unprivileged browser (I haven't ruled Java out also) was the doorway, it would explain a few other things as well.  Apologies again.

**Comment by arolson101** [comment]
I think this issue is dead.  Please update if more support is needed!
